Mathlib/Data/Finset/Card.lean

```diff
@@ -539,6 +539,10 @@ theorem card_le_one_iff : s.card ≤ 1 ↔ ∀ {a b}, a ∈ s → b ∈ s → a
   tauto
 #align finset.card_le_one_iff Finset.card_le_one_iff
 
+theorem card_le_one_iff_subsingleton : s.card ≤ 1 ↔ Subsingleton s := by
```

Comment:
Isn't this the same as `card_le_one` above? Can we remove that lemma and use this one instead?

Reply:
Oh, there's a coercion on the RHS! In that case, perhaps
```suggestion
theorem card_le_one_iff_subsingleton_coeSort : s.card ≤ 1 ↔ Subsingleton s := by
```

Reply:
Then I think a better name might be:
```suggestion
theorem card_le_one_iff_subsingleton_coe : s.card ≤ 1 ↔ Subsingleton s := by
```
or
```suggestion
theorem card_le_one_iff_subsingleton_coeSort : s.card ≤ 1 ↔ Subsingleton s := by
```
(I don't know if either or both have precedent)

Reply:
OK I've added [a commit](https://github.com/leanprover-community/mathlib4/pull/7131/commits/9ce8a35d1f049e38360daabcd75bfe3a8a327c6a) along these lines.